tests/test_glean.py

```diff
@@ -854,82 +854,6 @@ def test_metadata_multiple_pings(self, mock_get_repos, config):
                     == "seconds"
                 )
 
-    # This is not a stubbed test, if it fails check if there have been changes with probe-scraper
```

Comment:
There were in fact changes to repositories.yaml as I'm trying to get rid of these pings: https://github.com/mozilla/probe-scraper/commit/28e8bf0b0b937041af0e60098e488172f818d386